### PR TITLE
Fixed download buttons for mobile devices.

### LINKED ISSUE #290: Apple App Store / Google Play Store links dead
Hey !
Thanks a lot for this great tool. It makes me gain so much time when inviting people.

I noticed the links to the Apple App Store & Google Play Store are dead. 
In DownloadAppStore.vue, DownloadAndroid.vue & DownloadLinux.vue, the RouterLink component cannot take users to external urls, it needs to be an "a" tag :
```html
<a :href="to" target="_blank">
```
instead of
```html
<RouterLink :to="to">
```

Furthermore, the links take you to the **webpage** of the store, not the actual store itself.
I think you can use :
```vue.js
<DownloadAppStore to="itms-apps://geo.itunes.apple.com/developer/swiftfin/id1604098728" />
```
and
```vue.js
<DownloadAndroid to="market://details?id=dev.jdtech.jellyfin" />
```
in Download.vue.
I tested the Android one, but I don't have an apple device to test the App store one. If anyone reads this and has an apple device, it would be nice if you could test this link and say if this link works, and indicate which iOS version you have :
itms-apps://geo.itunes.apple.com/developer/swiftfin/id1604098728

Thanks again for your work 🚀

### PR DESCRIPTION
**Pull request by JamsRepos**
This rectifies the incorrect logic within #290.

Closes #290

### CODE REVIEW

File: apps/wizarr-frontend/src/modules/help/components/Jellyfin/Download.vue
[the remark this answers is not in the document]
Comment (JamsRepos) [reply apps/wizarr-frontend/src/modules/help/components/Jellyfin/Download.vue:1]: Yeah, @Moisty145 tested for Android and I tested for Apple. 

The only thing to note though is it doesn't work on Apple for other browsers, Safari is the only supported one. Was toying with the idea of adding a link underneath also.

[the remark this answers is not in the document]
Comment (realashleybailey) [reply apps/wizarr-frontend/src/modules/help/components/Jellyfin/Download.vue:1]: Can we not keep the deep link ??

[the remark this answers is not in the document]
Comment (JamsRepos) [reply apps/wizarr-frontend/src/modules/help/components/Jellyfin/Download.vue:1]: > Can we not keep the deep link ??

Sure - would just redirect them to the link of the app store.

[the remark this answers is not in the document]
Comment (realashleybailey) [reply apps/wizarr-frontend/src/modules/help/components/Jellyfin/Download.vue:1]: Why did we switch the deep link to the http link anyway?

[the remark this answers is not in the document]
Comment (JamsRepos) [reply apps/wizarr-frontend/src/modules/help/components/Jellyfin/Download.vue:1]: Issue #290

**Pull request merged**